lnrpc/routerrpc/router_backend.go

```diff
@@ -383,6 +383,7 @@ func (r *RouterBackend) parseQueryRoutesRequest(in *lnrpc.QueryRoutesRequest) (
 		DestCustomRecords: record.CustomSet(in.DestCustomRecords),
 		CltvLimit:         cltvLimit,
 		DestFeatures:      destinationFeatures,
+		BlindedPayment:    blindedPmt,
```

Comment:
Should this change be picked into a new commit?

Reply:
100%